packages/interfaces/src/IAppliance.js

```diff
@@ -4,6 +4,11 @@ import {
 } from '@tvkitchen/base-errors'
 
 class IAppliance {
+	// A simple, replaceable logger for use by the Appliance.
+	logger = {
+		log: (level, message) => console.log(`${level}: ${message}`), // eslint-disable-line no-console
```

Comment:
A thought here, I think it will be necessary to provide a list of official TVKitchen log levels  in our `base-constants`.

Given that, I wonder if it would be wise to create `.error` `.info` and `.warn` methods which call log and pass the appropriate constant, so that Appliance implementations don't need to load in those constants themselves (although that package will necessarily be a dependency already since Appliances will need access to the dataTypes constants)

Reply:
@reefdog you approved but also gave a 👍 -- do you think I should bake in .error .info .warn and import log levels, as part of this PR or no?

Reply:
Haha I was moving too fast… I assumed this PR would be the first step and then the suggested approach would be a separate iteration.

I do think error/info/warn methods and the constantized log levels are a good idea and should be added ASAP, and yeah I suppose it makes sense to pivot this PR to cover them! I retract my approval!

Reply:
Added this -- CI fails since base-constants doesn't have log levels in npm yet, but once that is merged I'll do a release.